README.md

```diff
@@ -1,16 +1,15 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/CeresDB/ceresdb-client-go.svg)](https://pkg.go.dev/github.com/CeresDB/ceresdb-client-go)
+
 # [CeresDB](https://github.com/CeresDB/ceresdb) Golang Client
 
 ## Introduction
 The golang client for CeresDB.
 CeresDB is a high-performance, distributed, schema-less, cloud native time-series database that can handle both time-series and analytics workloads.
 
 ## Support features
-- Manage
-  - creating and alter table
-- Query data
-  - using sql
-- Write data
-  - Point build
+- DDL, such as create/alter table
+- Query using SQL
+- Write using gRPC
```

Comment:
grpc is an internal implementation, and SQL is also grpc, it is better to use the time series data interface?